packages/x-adapter/src/http-clients/__tests__/fetch.http-client.spec.ts

```diff
@@ -0,0 +1,121 @@
+import { HttpClient } from '../../types/http-client.types';
+import { koFetchMock, okFetchMock } from '../__mocks__/fetch.mock';
+
+describe('fetch httpClient testing', () => {
+  const endpoint = 'https://api.empathy.co/search';
+
+  let fetchHttpClient: HttpClient;
+
+  beforeEach(async () => {
+    fetchHttpClient = (await import('../fetch.http-client')).fetchHttpClient;
```

Comment:
I've refactored this so the module is reset in every test case. With the current test there is no problem, but the `requestAbortControllers` dictionary was not being reset and it may cause some unexpected issues.